src/core/mdn-annotation.js

```diff
@@ -86,8 +86,8 @@ function buildBrowserSupportTable(support) {
   let innerHTML = "";
   function addMDNBrowserRow(browserId, yesNoUnknown, version) {
     const displayStatus = yesNoUnknown === "Unknown" ? "?" : yesNoUnknown;
-    const classList = `${browserId} ${yesNoUnknown.toLowerCase()}`;
-    const browserRow = `
+    const classList = hyperHTML `${browserId} ${yesNoUnknown.toLowerCase()}`;
+    const browserRow = hyperHTML `
```

Comment:
This is good but but need more changes because `innerHTML += browserRow` will be broken. I suggest replacing `innerHTML` with an array and push to it instead.